web-components/src/components/combobox/ComboBox.ts

```diff
@@ -776,8 +776,10 @@ export namespace ComboBox {
 
     handleInput(event: Event) {
       const inputValue = (event.target as HTMLInputElement).value;
-      this.inputValue = inputValue;
-      this.notifyInputValueChanged(inputValue);
+      if (inputValue !== " ") {
```

Comment:
what if inputValue has 2 spaces? trim the spaces
 and check

Reply:
handled trimming of input value